app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,15 @@
   <head>
     <title><%= content_for(:page_title) || "Samson" %></title>
     <%= stylesheet_link_tag    "application", media: "all" %>
+    <% Samson::Hooks.plugins.map(&:stylesheets).flatten.each do |css| %>
+      <%= stylesheet_link_tag asset_path(css) %>
+    <% end %>
+
     <%= javascript_include_tag "application" %>
+    <% Samson::Hooks.plugins.map(&:javascripts).flatten.each do |javascript| %>
```

Comment:
Something like this may be helpful: http://edgeguides.rubyonrails.org/engines.html#separate-assets-precompiling

There must be a standard way to adds assets from a rails engine - they may even be precompiled in by default?